### PR TITLE
Create identities.cs-e2e.ts

### PR DESCRIPTION
**Pull request by cfaur09**
## Proposed Changes
- Create identities.cs-e2e.ts

## How to test
- npm run test:cs-e2e identities.cs-e2e.ts

### CODE REVIEW

File: src/test/chain-simulator/transactions.cs-e2e.ts [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+    it('should support pagination', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/transactions?from=0&size=1`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toStrictEqual(1);
+    });
+
+    it('should return transactions with sender filter applied', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/transactions?sender=${config.aliceAddress}`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toBeGreaterThan(1);
```
Comment (bogdan-rosianu) [inline src/test/chain-simulator/transactions.cs-e2e.ts:25]: check that all the transactions contain the provided sender

File: src/test/chain-simulator/transactions.cs-e2e.ts [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+    });
+
+    it('should return transactions with receiver filter applied', async () => {
+      const receiver = 'erd1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq6gq4hu';
+      const response = await axios.get(`${config.apiServiceUrl}/transactions?receiver=${receiver}`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toBeGreaterThan(1);
```
Comment (bogdan-rosianu) [inline src/test/chain-simulator/transactions.cs-e2e.ts:32]: check the receiver field of the results

File: src/test/chain-simulator/transactions.cs-e2e.ts [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+      const response = await axios.get(`${config.apiServiceUrl}/transactions?senderShard=${senderShard}`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toBeGreaterThanOrEqual(1);
+    });
+
+    it('should return transactions with receiverShard filter applied', async () => {
+      const receiverShard = 0;
+      const response = await axios.get(`${config.apiServiceUrl}/transactions?receiverShard=${receiverShard}`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toBeGreaterThanOrEqual(1);
```
Comment (bogdan-rosianu) [inline src/test/chain-simulator/transactions.cs-e2e.ts:46]: check the receiver shard of the results

File: src/test/chain-simulator/transactions.cs-e2e.ts [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+      expect(response.status).toBe(200);
+      expect(response.data.length).toStrictEqual(1);
+    });
+
+    it('should return transactions with hashes filter applied', async () => {
+      const transaction = await axios.get(`${config.apiServiceUrl}/transactions?size=2`);
+      const hashes = [transaction.data[0].txHash, transaction.data[1].txHash];
+      const response = await axios.get(`${config.apiServiceUrl}/transactions?hashes=${hashes}`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toStrictEqual(2);
```
Comment (bogdan-rosianu) [inline src/test/chain-simulator/transactions.cs-e2e.ts:62]: check the hashes of the results

File: src/test/chain-simulator/transactions.cs-e2e.ts [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+    it('should return transactions with status filter applied', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/transactions?status=success`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toBeGreaterThanOrEqual(1);
+    });
+
+    it('should return transactions with function filter applied', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/transactions?function=scDeploy`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toBeGreaterThanOrEqual(1);
```
Comment (bogdan-rosianu) [inline src/test/chain-simulator/transactions.cs-e2e.ts:74]: check the function of the results

File: src/test/chain-simulator/transactions.cs-e2e.ts [hunk 6]
```diff
@@ -0,0 +1,291 @@
...
+    });
+
+    it('should return transactions with round filter applied', async () => {
+      const transaction = await axios.get(`${config.apiServiceUrl}/transactions?size=1`);
+      const round = transaction.data[0].round;
+      const response = await axios.get(`${config.apiServiceUrl}/transactions?round=${round}`);
+      expect(response.status).toBe(200);
+      expect(response.data.length).toBeGreaterThanOrEqual(1);
```
Comment (bogdan-rosianu) [inline src/test/chain-simulator/transactions.cs-e2e.ts:82]: check the round of the results

File: src/test/chain-simulator/transactions.cs-e2e.ts (hunk 1 above)
Comment (cfaur09) [reply src/test/chain-simulator/transactions.cs-e2e.ts:25]: done

File: src/test/chain-simulator/transactions.cs-e2e.ts (hunk 3 above)
Comment (cfaur09) [reply src/test/chain-simulator/transactions.cs-e2e.ts:58]: done

File: src/test/chain-simulator/transactions.cs-e2e.ts (hunk 4 above)
Comment (cfaur09) [reply src/test/chain-simulator/transactions.cs-e2e.ts:82]: done

File: src/test/chain-simulator/transactions.cs-e2e.ts (hunk 5 above)
Comment (cfaur09) [reply src/test/chain-simulator/transactions.cs-e2e.ts:102]: done

File: src/test/chain-simulator/transactions.cs-e2e.ts (hunk 6 above)
Comment (cfaur09) [reply src/test/chain-simulator/transactions.cs-e2e.ts:114]: done

**Review by bogdan-rosianu — APPROVED** [review]

**Review by dragos-rebegea — COMMENTED** [review]
for these we should also include the assets-cdn to check that brandings. This can be done in this PR or a separate one

**Review by dragos-rebegea — APPROVED** [review]

**Pull request merged**